migrations/Version202203151417562260_taoQtiTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use common_ext_Extension as Extension;
+use common_ext_ExtensionsManager as ExtensionsManager;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version202203151417562260_taoQtiTest extends AbstractMigration
+{
+    private const CONFIG_FILE = 'testRunner';
+
+    public function getDescription(): string
+    {
+        return 'Set up default config for test-runner review panel';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $extension = $this->getExtension();
+
+        $config = $extension->getConfig(self::CONFIG_FILE);
+        $config['plugins']['review']['reviewLayout'] = 'default';
+        $config['plugins']['review']['displaySectionTitles'] = true;
+        $config['plugins']['review']['displayItemTooltip'] = false;
```

Comment:
**nitpick:** Not sure this is safe. Unfortunately, the previous pull requests for the new layout did not enforce the settings. But meanwhile, other extensions might have done it, and this script will overwrite the config.
While this script can fix this miss, it would need to do it using a smarter way: if the config is not set, then the default value must be set, otherwise, it must remain untouched.

**suggestion:** The script should rather do something like:
```suggestion
        // Fix the config as the previous migrations were not adding it.
        // Set the default config if it does not exist.
        if (!array_key_exists('review', $config['plugins'])) {
            $config['plugins']['review'] = [];
        }
        if (!array_key_exists('reviewLayout', $config['plugins']['review'])) {
            $config['plugins']['review']['reviewLayout'] = 'default';
        }
        if (!array_key_exists('displaySectionTitles', $config['plugins']['review'])) {
            $config['plugins']['review']['displaySectionTitles'] = true;
        }
        
        // Set the new config entry
        $config['plugins']['review']['displayItemTooltip'] = false;
```

Reply:
Thanks, it's the best, for some reason I didn't think of it